.eslint-dictionary.json

```diff
@@ -146,6 +160,7 @@
   "foreach",
   "formatter",
   "frontend",
+  "frontendi",
```

Comment:
`packages/amplify-frontend-ios/index.js`
Capitalize the `i` in `iOS`, and remove this from the dictionary